documentation/tutorial/04-deploy-superset.md

```diff
@@ -0,0 +1,59 @@
+# Deploying Charmed Superset
+
+This is part of the
+[Charmed Superset Tutorial]().
+Please refer to this page for more information and the overview of the content.
+
+Charmed Superset has 3 primary functions:
+1. Web server and UI
+2. Worker for handling asynchronous queries
+3. Beat scheduler for scheduling tasks
+
+For the purposes of our tutorial we will only need the `Web server and UI`.
+
+## Deploy Charmed Superset Server
+There are multiple channels of Charmed Superset we can deploy - these can be found on [Charmhub](https://charmhub.io/superset-k8s). If none is specified, `latest/stable` will be deployed.
+
+```bash
+juju deploy superset-k8s
+```
+
+## Add PostgreSQL and Redis relations
+A charm relation is an integration between applications which passes necessary configurations to automate the integration. We'll now add relations between Charmed Superset and Charmed PostgreSQL and Charmed Redis as follows:
+
+```bash
+juju relate superset-k8s postgresql-k8s
+juju relate superset-k8s redis-k8s
+```
+
```

Comment:
Add here a note box saying this relations addition may take up to five minutes to complete.